x-pack/plugins/fleet/public/lazy_custom_logs_assets_component.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import url from 'url';
+import { stringify } from 'querystring';
+
+import React, { lazy } from 'react';
+import { encode } from 'rison-node';
+
+import type { PackageAssetsComponent } from './types';
+import { CustomAssetsAccordion } from './components/custom_assets_accordion';
+import type { CustomAssetsAccordionProps } from './components/custom_assets_accordion';
+
+export const LazyCustomLogsAssetsComponent = lazy<PackageAssetsComponent>(async () => {
+  // Filter to only custom logs, which by default appear in the "generic" dataset
+  const logStreamQuery = 'data_stream.dataset: "generic"';
```

Comment:
that is by default, but it is user-configurable per policy so we can't guarantee what the eventual dataset will end up being. let's leave off any querying for this view and leave it up to the user to filter to the custom logs they want to view